defaults/main.yml

```diff
@@ -23,12 +23,31 @@ sonar_file: "{{ sonar_name }}.zip"
 sonar_url: 'https://binaries.sonarsource.com/CommercialDistribution/sonarqube-enterprise/{{ sonar_file }}'
 sonar_home: /opt/sonar
 sonar_directory: "{{ sonar_home }}/sonarqube-{{ sonar_version }}"
-sonar_web_javaopts: "-Xmx1024m -Xms128m -XX:+HeapDumpOnOutOfMemoryError"
-sonar_web_javaadditionalopts: False
-sonar_search_javaadditionalopts: "-Dbootstrap.system_call_filter=false"
+dd_jvm_opts: >
+  -Ddd.trace.report-hostname=true
+  -Ddd.jmxfetch.jetty.enabled=true
+  -Ddd.trace.global.tags=env:management,application:{{ sonar_name }},service:{{ sonar_name }} -Ddd.http.client.tag.query-string=true
+  -Ddd.http.server.tag.query-string=true
+  -Ddd.logs.injection=true
+  -Ddd.trace.analytics.enabled=true
+
+sonar_web_javaopts: "-javaagent:/etc/datadog-agent/dd-java-agent.jar -Xmx1024m -Xms128m -XX:+HeapDumpOnOutOfMemoryError"
+sonar_web_javaadditionalopts: "-Ddd.service.name=web.{{ sonar_name }} {{ dd_jvm_opts }} -Dbootstrap.system_call_filter=false"
```

Comment:
Any reason you're using `-Ddd.service.name=web.{{ sonar_name }}`, which will be interpolated to `dd.service.name=web.sonarqube-enterprise-6.2`? Consider a new variable like `dd_service_name` so you aren't re-using a variable that isn't meant for defining the datadog service name.

I'd also re-think the the prefix `web/es/ce`, it will be confusing to the larger tech org. Maybe a suffix?

Reply:
Is this what you're referring to in the PR description:
> The service name won't register for sonarqube's database (postgresql and elasticsearch) that're in the instance. I cannot change their name in the reporting.

If so, best to leave a note about that in this file so others will understand why the decision to name the services this way was made